lib/task-data/tasks/secure-erase-drive.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     injectableName: 'Task.Drive.SecureErase',
     implementsTask: 'Task.Base.SecureErase',
     options: {
-        eraseSettings: []
+        eraseSettings: [],
+        baseUri: '{{ file.server }}'
```

Comment:
I agree with @cgx027 's suggestion.
Even more, we can add a new template item like "{{api.notification}}"